static/src/js/__tests__/App.test.js

```diff
@@ -309,8 +309,16 @@ describe('App component', () => {
   })
 
   // https://stackoverflow.com/questions/66667827/react-testing-library-to-cover-the-lazy-load/66690463
-  test('renders lazy component', () => {
+  test('renders loaded lazy components', () => {
     setup()
     expect(screen.getByTestId('mocked-map-container')).toBeInTheDocument()
+
+    // Lazy loaded routes not immediately loaded onto `App.js`
+    expect(screen.queryByTestId('mocked-admin')).not.toBeInTheDocument()
+    expect(screen.queryByTestId('mocked-contact-info')).not.toBeInTheDocument()
```

Comment:
Split this out and check that each gets mocked if the route is on that page instead